src/components/NatureOfBusiness/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@bcrs-shared-components/nature-of-business",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@bcrs-shared-components/interfaces": "^1.0.22",
```

Comment:
I'm hoping Lerna will automatically update this import to v23 (which is the version with the new interfaces). We'll see.

Reply:
Maybe it doesn't matter, because this component expects to find the interfaces in "@/interfaces", which exists here and will exist in the parent web app.
So I dunno what this import is for, then. Perhaps to make this component more standalone (ie, don't depend on parent to provide the interfaces)? 🤷‍♂️